### PR TITLE
Added lsblk and blkid for issue #113

### PR DESCRIPTION
**Pull request by JoeyPetersen**
Here are the proposed additions.

### CODE REVIEW

File: configsnap [hunk 1]
```diff
@@ -571,6 +571,10 @@ if os.path.exists('/etc/multipath.conf'):
 if os.path.exists('/sbin/powermt'):
     run.run_command(['powermt', 'display', 'dev=all'],
                     'powermt', fail_ok=True, sort=False, stdout=False)
+if os.path.exists('/usr/bin/lsblk')
```
Comment (rhodesn) [inline configsnap]: Missing a `:` on the end of here.

File: configsnap [hunk 2]
```diff
@@ -571,6 +571,10 @@ if os.path.exists('/etc/multipath.conf'):
...
+    run.run_command(['lsblk'], 'lsblk', fail_ok=True, sort=False)
+if os.path.exists('/usr/bin/blkid')
```
Comment (rhodesn) [inline configsnap]: Missing `:`.

**Review by rhodesn — CHANGES_REQUESTED** [review]
Thanks for the PR - left a couple minor comments.

File: configsnap (hunk 1 above)
Comment (JoeyPetersen) [reply configsnap]: I amended my commit and fixed this.

File: configsnap (hunk 2 above)
Comment (JoeyPetersen) [reply configsnap]: I amended my commit and fixed this.

**Comment by rhodesn** [comment]
Thanks!

**Pull request merged**